src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -178,6 +179,29 @@ public Collection<MergeReviewConceptVersions> getMergeReviewConflictingConcepts(
 		return conflicts.values();
 	}
 
+	private boolean isTargetConceptVersionBehind(Concept sourceVersion, Concept targetVersion) {
+		Set<Integer> sourceEffectiveTimes = new HashSet<>();
+		sourceEffectiveTimes.add(sourceVersion.getReleasedEffectiveTime());
+		sourceEffectiveTimes.addAll(sourceVersion.getRelationships().stream().map(Relationship::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getDescriptions().stream().map(Description::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getDescriptions().stream().map(Description::getAssociationTargetMembers).flatMap(List::stream).map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getAllOwlAxiomMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getInactivationIndicatorMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		sourceEffectiveTimes.addAll(sourceVersion.getAssociationTargetMembers().stream().map(ReferenceSetMember::getReleasedEffectiveTime).collect(Collectors.toList()));
+		while (sourceEffectiveTimes.remove(null)) ;
+
+		Set<Integer> targetEffectiveTimes = new HashSet<>();
+		targetEffectiveTimes.add(targetVersion.getReleasedEffectiveTime());
+		targetEffectiveTimes.addAll(targetVersion.getRelationships().stream().map(Relationship::getReleasedEffectiveTime).collect(Collectors.toList()));
+		targetEffectiveTimes.addAll(targetVersion.getDescriptions().stream().map(Description::getReleasedEffectiveTime).collect(Collectors.toList()));
```

Comment:
The targetEffectiveTimes is a set already so we could collect them as set instead of list. Also we can add isReleased() filter to avoid null values.

Reply:
Thanks for the suggestion @CoderMChu. I have updated the method and done this.